subprojects/docs/src/docs/release/notes.md

```diff
@@ -279,6 +279,14 @@ In Gradle 7.0 we moved the following classes or methods out of incubation phase.
         - org.gradle.process.JavaExecSpec.getMainClass()
         - org.gradle.process.JavaExecSpec.getMainModule()
         - org.gradle.process.JavaExecSpec.getModularity()
+    - Testing
+        - org.gradle.api.tasks.testing.Test.getStableClasspath
+        - org.gradle.api.plugins.JavaBasePlugin.COMPILE_CLASSPATH_PACKAGING_SYSTEM_PROPERTY
```

Comment:
this one is not really "Testing", but more general Java plugin (first category above).

Reply:
Right, indeed in my mind it was associated to testing because we added it in the context of testing on Windows, but it's more than that.